src/main/java/seedu/duke/commands/OpenLessonLinkCommand.java

```diff
@@ -16,62 +16,59 @@
 import java.util.Scanner;
 
 import static seedu.duke.commands.DeleteLessonCommand.getLessonName;
+import static seedu.duke.common.Messages.MESSAGE_LESSON_TO_OPEN_LINK;
+import static seedu.duke.common.Messages.MESSAGE_OPENED_LESSON_LINK;
+import static seedu.duke.common.Messages.MESSAGE_UNABLE_TO_OPEN_LINK;
 
 public class OpenLessonLinkCommand extends Command {
-    public static final String MESSAGE_OPEN_LESSON_LINK = "Which lesson’s link would you like to open?";
-    public static final String OPEN_LESSON_LINK_FORMAT = "Opening %s link in browser." + System.lineSeparator();
-    public static final String MESSAGE_CANNOT_OPEN_LESSON_LINK = "Cannot open lesson link" + System.lineSeparator();
     public static final String LINUX_COMMAND = "xdg-open ";
     private final Scanner commandLineReader = new Scanner(System.in);
```

Comment:
Maybe this line and line 37 could also make use of ui.readCommand()?

Reply:
Ok. Will replace all of it.